DEPS

```diff
@@ -357,7 +358,8 @@ deps = {
    Var('dart_git') + '/root_certificates.git' + '@' + Var('dart_root_certificates_rev'),
 
   'src/third_party/skia':
-   Var('skia_git') + '/skia.git' + '@' +  Var('skia_revision'),
+   #Var('skia_git') + '/skia.git' + '@' +  Var('skia_revision'),
+   Var('liguisheng_skia_git') + '@' + '99cac3c3ace05969d7bcf62578de274cca071df9',
```

Comment:
We cannot consider this patch because it changes the Skia upstream.

Reply:
C function setGpuRenderDisabled  in third_party/skia,  skia PR  https://github.com/google/skia/pull/76